nd4j/RaspberryPi.md

```diff
@@ -0,0 +1,46 @@
+Please follow following instructions to build nd4j for raspberry PI:
+
+1. download cross compilation tools for Raspberry PI
+
+    ```
+    $ apt-get/yum install git rsync cmake
+    $ mkdir $HOME/raspberrypi
+    $ cd $HOME/raspberrypi
+    $ git clone git://github.com/raspberrypi/tools.git
+    $ export PATH=$PATH:$HOME/raspberrypi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin
+    $ mkdir $HOME/raspberrypi/rootfs
+    (Replace the ip address in the following command with the IP of a raspberry pi device)
+    $ rsync -rl --delete-after --safe-links pi@(192.168.1.PI):/{lib,usr} $HOME/raspberrypi/rootfs
+    $ cat > $HOME/raspberrypi/pi.cmake <<END
```

Comment:
Could you add that file to the repository instead?

Reply:
Not a big problem.

Reply:
Modified and modeled after Android.